lib/datadog/tracing/contrib/opensearch/integration.rb

```diff
@@ -18,8 +18,8 @@ class Integration
           register_as :opensearch, auto_patch: true
 
           def self.version
-            Gem.loaded_specs['opensearch-transport'] \
-              && Gem.loaded_specs['opensearch-transport'].version
+            Gem.loaded_specs['opensearch-ruby'] \
```

Comment:
Since the transport gem merges into the main gem, this is the changes required to have 3.x instrumented.